paasta_tools/marathon_tools.py

```diff
@@ -679,6 +690,7 @@ def format_marathon_app_dict(self) -> FormattedMarathonAppDict:
             'env': self.get_env(),
             'mem': float(self.get_mem()),
             'cpus': float(self.get_cpus()),
+            'gpus': float(self.get_gpus()),
```

Comment:
This will cause a big bounce, right?

Reply:
@solarkennedy Good catch! Also, marathon needs to be restarted to have the --enable_features gpu_resources option. I will make this conditional.